runtime/lang/menu_de_de.latin1.vim

```diff
@@ -1,312 +1,1267 @@
-menutrans &Open\.\.\.<Tab>:e		&�ffnen\.\.\.<Tab>:e
-menutrans Sp&lit-Open\.\.\.<Tab>:sp	In\ geteiltem\ &Fenster\ �ffnen\.\.\.<Tab>:sp
-menutrans Open\ Tab\.\.\.<Tab>:tabnew	In\ neuem\ &Tab\ �ffnen\.\.\.<Tab>:tabnew
-menutrans &New<Tab>:enew		&Neue\ Datei<Tab>:enew
-menutrans &Close<Tab>:close		S&chlie�en<Tab>:close
-menutrans &Save<Tab>:w			&Speichern<Tab>:w
-menutrans Save\ &As\.\.\.<Tab>:sav	Speichern\ &als\.\.\.<Tab>:sav
-menutrans &Print			&Drucken
-menutrans Sa&ve-Exit<Tab>:wqa		Speichern\ und\ Be&enden<Tab>:wqa
-menutrans E&xit<Tab>:qa			&Beenden<Tab>:qa
-
-if has("diff")
-    menutrans Split\ &Diff\ with\.\.\.	D&ifferenz\ in\ geteiltem\ Fenster\ mit\.\.\.
-    menutrans Split\ Patched\ &By\.\.\.	&Patch\ in\ geteiltem\ Fenster\ mit\.\.\.
+" File menu
+menutrans &File &Datei
+" File menuitems {{{1
+menutrans &Open\.\.\.<Tab>:e &�ffnen\.\.\.<Tab>:e
+menutrans Sp&lit-Open\.\.\.<Tab>:sp In\ geteiltem\ &Fenster\ �ffnen\.\.\.<Tab>:sp
+menutrans Open\ Tab\.\.\.<Tab>:tabnew In\ neuem\ &Tab\ �ffnen\.\.\.<Tab>:tabnew
+menutrans &New<Tab>:enew &Neue\ Datei<Tab>:enew
+menutrans &Close<Tab>:close S&chlie�en<Tab>:close
+menutrans &Save<Tab>:w &Speichern<Tab>:w
+menutrans Save\ &As\.\.\.<Tab>:sav Speichern\ &als\.\.\.<Tab>:sav
+menutrans Split\ &Diff\ With\.\.\. D&ifferenz\ in\ geteiltem\ Fenster\ mit\.\.\.
+menutrans Split\ Patched\ &By\.\.\. &Patch\ in\ geteiltem\ Fenster\ mit\.\.\.
+menutrans &Print &Drucken
+menutrans Sa&ve-Exit<Tab>:wqa Speichern\ und\ Be&enden<Tab>:wqa
+menutrans E&xit<Tab>:qa &Beenden<Tab>:qa
+" }}}
+
+" Edit menu
+menutrans &Edit &Editieren
+" Edit menuitems {{{1
+menutrans &Undo<Tab>u Z&ur�ck<Tab>u
+menutrans &Redo<Tab>^R Vo&r<Tab>^R
+menutrans Rep&eat<Tab>\. &Wiederholen<Tab>\.
+menutrans Cu&t<Tab>"+x &Ausschneiden<Tab>"+x
+menutrans &Copy<Tab>"+y &Kopieren<Tab>"+y
+menutrans &Paste<Tab>"+gP Ein&f�gen<Tab>"+gP
+menutrans Put\ &Before<Tab>[p Da&vor\ einf�gen<Tab>[p
+menutrans Put\ &After<Tab>]p Da&nach\ einf�gen<Tab>]p
+menutrans &Delete<Tab>x &L�schen<Tab>x
+menutrans &Select\ All<Tab>ggVG Alles\ &markieren<Tab>ggVG
+menutrans &Find\.\.\. &Suchen\.\.\.	" conflicts with Edit.Find
+menutrans Find\ and\ Rep&lace\.\.\. Suchen\ und\ &Ersetzen\.\.\.
+" menutrans &Find<Tab>/ TRANSLATION\ MISSING
+" menutrans Find\ and\ Rep&lace<Tab>:%s TRANSLATION\ MISSING
+" menutrans Find\ and\ Rep&lace<Tab>:s TRANSLATION\ MISSING
+menutrans Settings\ &Window E&instellungen\.\.\.
+menutrans Startup\ &Settings &Starteinstellungen
+
+" Edit/Global Settings
+menutrans &Global\ Settings &Globale\ Einstellungen
+" Edit.Global Settings menuitems and dialogs {{{2
+menutrans Toggle\ Pattern\ &Highlight<Tab>:set\ hls! &Hervorhebungen\ ein-\ und\ ausschalten<Tab>:set\ hls!
+menutrans Toggle\ &Ignoring\ Case<Tab>:set\ ic! Gro�schreibung\ &ignorieren\ oder\ benutzen<Tab>:set\ ic!
+menutrans Toggle\ &Showing\ Matched\ Pairs<Tab>:set\ sm! Anzeige\ des\ passenden\ &Symbols\ ein-\ und\ ausschalten<Tab>:set\ sm!
+menutrans &Context\ Lines &Zusammenhang
```

Comment:
not sure about `Zusammenhang` perhaps just `Kontext`

Reply:
> not sure about `Zusammenhang` perhaps just `Kontext`

Thanks, I've replaced it in the new commit.